build/Dockerfile.builder

```diff
@@ -29,5 +29,4 @@ RUN yum -y install clang rpm-build llvm-devel
 
 # enable EPEL and install cpuid
 RUN yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
-RUN yum install -y cpuid
```

Comment:
since previous line, which instals epel, is also moved to builder, can we delete it also?

Reply:
removed as your suggestion.